ttnn/cpp/ttnn/operations/experimental/transformer/rotary_embedding_llama/rotary_embedding_llama.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "rotary_embedding_llama.hpp"
+
+#include "device/rotary_embedding_llama_device_operation.hpp"
+
+namespace ttnn::operations::experimental::transformer {
+
+Tensor RotaryEmbeddingLlamaOperation::operator()(
+    const Tensor &input_tensor,
+    const Tensor &cos,
```

Comment:
I'd use cos_cache and sin_cache. using namespace std might be somewhere near  don't want to see weird errors in future :D

Reply:
100% actually replaced it in the other one, but forgot here. Thank you